common/src/main/kotlin/org/wycliffeassociates/otter/common/domain/content/TakeActions.kt

```diff
@@ -82,6 +85,27 @@ class TakeActions @Inject constructor(
             }
     }
 
+    fun saveChunk(
```

Comment:
The purpose of TakeActions are to process takes with plugins, not to add takes to the workbook (which only needs to push a take into the take relay with insert take. As such, I don't feel like this functionality belongs here, and the function here named the way that it does can give the confusing impression that takes that are recorded, edited, or marked are not "saved" and need to use this to save changes.

This functionality should be part of Narration domain related use case classes internal to a Narration package.